netmap/netmap_contract.go

```diff
@@ -685,9 +703,9 @@ func setConfig(ctx storage.Context, key, val any) {
 }
 
 func cleanup(ctx storage.Context, epoch int) {
-	balanceContractAddr := storage.Get(ctx, balanceContractKey).(interop.Hash160)
-	contract.Call(balanceContractAddr, cleanupEpochMethod, contract.All, epoch)
-
-	containerContractAddr := storage.Get(ctx, containerContractKey).(interop.Hash160)
```

Comment:
How about deleting these keys from DB on update and marking these options as unused and not requiring any checks on deploy?

Reply:
missed that. but we need to decide: migration or removal

Reply:
Old keys should be removed, they're useless now. And given that they're a part of contract internal storage, they should be migrated into new ones.

Reply:
done